backend/services/location.service.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable func-names */
+const { Location } = require('../models/dictionaries/location.model');
+const DatabaseError = require('../errors/database.error');
+
+const LocationService = {};
+
+function validateLocations(locationsarray){
+
+  if(!Array.isArray(locationsarray)){
+    throw "locations array must be of type array"
+  }
+  if(locationsarray.length <= 0){
+    throw "locations array must have at least one element";
+  }
+  if( !locationsarray.every(item => (typeof item === "string")) ){
+    throw "locations array must be an array of strings"
+  }
+  if( !locationsarray.every( (element) => { if(element){return true} } )){
+    throw "locations parameter is an array of non-empty strings"
```

Comment:
please improve the message to "locations array should not contain empty strings"

Reply:
messages improved as stated